wordpress-mobile/WordPress-Android
# Fix empty view for saved posts list

Fixes #7808 

![screenshot_1527158894](https://user-images.githubusercontent.com/2261188/40481395-a46e60ec-5f51-11e8-8a0f-20331b6dbec3.png)

To test:
1. Go to reader
2. Go to Saved posts list
3. Make sure you don't have any posts saved
4. Notice the empty view is shown
